plugins/lookup/merge_variables.py

```diff
@@ -61,6 +62,13 @@
         ini:
           - section: merge_variables_lookup
             key: override
+      groups:
+        description:
+          - Search for variables accross hosts that belong to the given groups. This allows to collect configuration pieces
+             accross different hosts (for example a service on a host with its database on another host).
+        type: list
+        elements: str
+        version_added: 8.4.0
```

Comment:
```suggestion
        version_added: 8.5.0
```
8.4.0 is already released.